server/src/api/resources/trip/trip.controller.js

```diff
@@ -137,3 +144,37 @@ export const repeatTrip = (req, res) => {
   delete updatedRequest.body.id
   createTrip(updatedRequest, res)
 }
+
+export const uploadPics = (req, res) => {
+  const id = req.params.id
+  cloudinary.v2.uploader.upload(req.body.image, (err, result) => {
```

Comment:
Everything here looks good, except maybe we should make sure `req.body.image` has a value before we pass it to the cloudinary API call. Or do something with the `err` response inside the callback?

Reply:
changed it to err 500 and made the change below